tests/dummy/app/controllers/index.js

```diff
@@ -8,7 +8,7 @@ export default Ember.Controller.extend({
 
   allCookies: computed(function() {
     let cookieService = this.get('cookies');
-    cookieService.write('now', new Date().getTime());
+    cookieService.write('now', new Date().getTime(), { domain: 'localhost' });
```

Comment:
Cool - that should be handled in a test case though. We shouldn't add it here as well as that couples the app to `localhost`.